### PR TITLE
admission: update TAG to 0.9.4

### PR DESCRIPTION
**Pull request by yokaze**
This version reverts fc56ace.

Signed-off-by: Daichi Sakaue <daichi-sakaue@cybozu.co.jp>

### CODE REVIEW

File: admission/TAG [hunk 1]
```diff
@@ -1 +1 @@
-0.9.3
+0.9.4
```
Comment (ysksuzuki) [inline admission/TAG:1]: Is this change intended to rebuild admission to reflect https://github.com/cybozu/neco-containers/pull/623/commits/6f831833c728ab9de5749f49d4f0b45ec9ad649a?

Comment (yokaze) [reply admission/TAG:1]: Yes, it is needed to complete cybozu-go/neco#1638.

**Review by ysksuzuki — APPROVED** [review]
